index.html

```diff
@@ -885,6 +885,14 @@ <h2>Verification Methods</h2>
 The value of the `controller` property MUST be a <a
 data-cite="INFRA#string">string</a> that conforms to the [[URL]] syntax.
                 </dd>
+                <dt><dfn class="lint-ignore">revoked</dfn></dt>
+                <dd>
+The `revoked` property is OPTIONAL. If provided, it MUST be an [[XMLSCHEMA11-2]]
+combined date and time string specifying when the <a>verification method</a>
+ceased to be used. Once the value is set, it is not expected to be updated, and
```

Comment:
`ceased to be used` doesn't seem right. It might continue to be used, by issuers and/or verifiers, if they don't see (or ignore) the `revoked` property value. Maybe `SHOULD cease to be used`?
```suggestion
SHOULD cease to be used. Once the value is set, it is not expected to be updated, and
```